src/splitEvery.js

```diff
@@ -0,0 +1,39 @@
+var _curry2 = require('./internal/_curry2');
+var _reduce = require('./internal/_reduce');
+
+
+/**
+ * Subdivides the input array into parts of size `n`.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig Number -> [a] -> [[a]]
+ * @param {Number} n The size of the batches.
+ * @param {Array} list The list to be divide into batches.
+ * @return {Array} The new list.
+ * @aka chunk
+ * @example
+ *
+ *      R.chunk(3, [5,3,8,1,2,9,7]);
+ *      //=> [ [ 5, 3, 8 ],  [ 1, 2, 9 ], [ 7 ] ]
+ */
+
+module.exports = _curry2(function(n, list) {
+  var idx = 0;
+  if (n < 1) {
+    return list.length === 0 ? [] : [list];
```

Comment:
That is not what I meant. I don't understand how wrapping the list in a list makes sense. I'd rather just do `if (n < 1) return []`. That doesn't do anything meaningful either but at least it doesn't pretend to do something that makes sense.

In any cases users are not going to get what they expect so an error _would_ be ideal. Such a thing could be expressed in a fancy argument validation system! ;)